src/core/object/watch/index.ts

```diff
@@ -886,3 +886,31 @@ export function unset(
 
 	engine.unset(obj, path, handlers);
 }
+
+/**
+ * The function mute object children
```

Comment:
Плохое описание, лучше вот так

```suggestion
 * Wraps the specified object with unwatchable proxy, i.e. any mutations of this proxy can’t be watched
```